src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1015,21 +1015,27 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, DiagnosticBag d
             }
             else
             {
+                var boundTypeArguments = BindTypeArguments(typeArguments, diagnostics, basesBeingResolved);
+                if (unconstructedType.IsGenericType && options.IsAttributeTypeLookup() && boundTypeArguments.Any(bta => bta.Type.IsTypeParameter()))
```

Comment:
>boundTypeArguments.Any(bta => bta.Type.IsTypeParameter() [](start = 90, length = 56)

This is not sufficient to determine if it *uses* type parameters.  You also need to exclude, for example, `List<T>`.

Reply:
thanks I improved the check and added this to the `OpenGenericTypeInAttributeWithGenericAttributeFeature` test